ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/transform/PubsubMessageToTableRow.java

```diff
@@ -4,11 +4,10 @@
 
 package com.mozilla.telemetry.ingestion.transform;
 
-import static com.mozilla.telemetry.ingestion.util.Attribute.CLIENT_ID;
-
 import com.google.cloud.bigquery.TableId;
 import com.google.pubsub.v1.PubsubMessage;
 import com.mozilla.telemetry.ingestion.io.BigQuery.Write.TableRow;
+import com.mozilla.telemetry.ingestion.util.Attribute;
```

Comment:
```suggestion
import com.mozilla.telemetry.ingestion.core.Attribute;
```

Now that we're sharing code between modules, I think we should try to keep the package naming in line with the directory naming, so everything in ingestion-core should be under subpackage `...intestion.core`. I don't feel strongly about whether we have an additional `util` subpackage beneath that or not. The `Attribute` class in particular feels important enough that it could live at the root of `core`.

Reply:
Sounds good, I'll also move sink classes to `ingestion.sink.*`, but leave ingestion-beam as-is because now (right before prod deploy) is not the time to introduce that change